I'm lost why this fails ... it has to be connected with the conversion 'solver' => 's' as #456 was working well (see: https://github.com/OpenModelica/OMPython/pull/456#issuecomment-4084889159); however, I'm currently without the right test environment to vary the setup; based on the pytest output, it gets a command line containing '-s=1.0' for stopTime = 1.0:

`
stopTime = 1.0
`
and
`
mod.setSimulationOptions(simOptions={"stopTime": stopTime})
`
result in
`
E           OMPython.ModelicaSystem.ModelicaSystemError: Error running command ['/tmp/tmpm6n2_nsv/M_getters', '-overrideFile=/tmp/tmpm6n2_nsv/M_getters_res_override.txt', '-r=/tmp/tmpm6n2_nsv/M_getters_res.mat', '-s=1.0']
`

--- a/OMPython/ModelicaSystem.py
+++ b/OMPython/ModelicaSystem.py
@@ -1059,7 +1059,7 @@ class ModelicaSystem:
             major, minor, patch = self.parse_om_version(self._version)
             if (major, minor, patch) >= (1, 26, 0):
                 for key, opt_value in self._simulate_options_override.items():
-                    k = "s" if key=="solver" else key
+                    k = "s" if key == "solver" else key
                     om_cmd.arg_set(key=k, val=str(opt_value))
                 override_content = (
                     "\n".join([f"{key}={value}" for key, value in self._override_variables.items()])

--- a/OMPython/ModelicaSystem.py
+++ b/OMPython/ModelicaSystem.py
@@ -1683,12 +1683,6 @@ class ModelicaSystem:
         override_file = self.getWorkDirectory() / f'{self._model_name}_override_linear.txt'
         override_file.write_text(override_content)
 
-        with open(file=override_file, mode="w", encoding="utf-8") as fh:
-            for key1, value1 in self._override_variables.items():
-                fh.write(f"{key1}={value1}\n")
-            for key2, value2 in self._linearization_options.items():
-                fh.write(f"{key2}={value2}\n")
-
         om_cmd.arg_set(key="overrideFile", val=override_file.as_posix())
 
         if self._inputs:

